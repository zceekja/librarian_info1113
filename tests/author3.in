add book book1.csv 111111
add book book1.csv 111118
add book book1.csv 111119
author J.K. Rowling