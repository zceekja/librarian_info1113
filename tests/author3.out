user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: Harry Potter 1 (J.K. Rowling).

user: Successfully added: Harry Potter 2 (J.K. Rowling).

user: Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)

user: 